app/validation/validators.py

```diff
@@ -201,6 +206,16 @@ def __call__(self, form, field):
             raise validators.ValidationError(self.message)
 
 
+class YearCheck:
+    def __init__(self, message=None):
+        self.message = message or error_messages['INVALID_DATE']
+
+    def __call__(self, form, field):
+        # Must be 4 digits long
```

Comment:
This is fine unless we ever want to allow people to enter dates between 0 and 999 AD... which I suspect isn't a requirement 👍

Reply:
To be honest I was tempted to ask if I could limit it to dates after 1900 but chose against that.